adapter/src/main/java/org/apache/kudu/client/KuduUtil.java

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Twilio, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kudu.client;
+
+public class KuduUtil {
+
+  public static byte[] encodePrimaryKey(final PartialRow row) {
```

Comment:
Helper class since encodePrimaryKey is package-private.

Reply:
you can just call `row.encodePrimaryKey()`
https://github.com/apache/kudu/blob/da2890729a51276b367783f06d792dbddffaf49d/java/kudu-client/src/main/java/org/apache/kudu/client/PartialRow.java#L1312-L1314

Reply:
D'oh. Done.